app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -1080,14 +1081,16 @@ class PipelineSampleReads extends React.Component {
         >
           {d_report}
         </div>
-        {this.props.admin && (
-          <SampleDetailsSidebar
-            visible={this.state.sampleDetailsSidebarVisible}
-            onClose={this.toggleSampleDetailsSidebar}
-            sample={this.props.sampleInfo}
-            onNameUpdate={newName => this.setState({ sample_name: newName })}
-          />
-        )}
+        {this.pipelineRun &&
```

Comment:
I'm fairly certain that the sample details always need to be visible (that's the current behavior) -- let's discuss with Jenn tomorrow